test/DotNetty.Codecs.Http2.Tests/Http2ConnectionRoundtripTest.cs

```diff
@@ -97,17 +97,15 @@ protected override void SetupBootstrap(Bootstrap bootstrap)
             bootstrap.Group(new DefaultEventLoopGroup()).Channel<LocalChannel>();
         }
 
-        protected override void StartBootstrap()
+        protected override async Task StartBootstrap()
```

Comment:
what is the purpose of this change?

Reply:
Some Codecs.Http2.Tests are very similar in terms of preparation for the test, but are re-written manually a bit differently (in example `void` in one test and `Task` in another). Moreover I needed the logs to see if the actual ConnectAsync operation completed on the client or server channel, and it took me multiple tries to emit logs for different classes.

Because of this poor copy-paste code, I introduced a single class `HttpToHttp2ConnectionHandlerTest` which has the API which also logs the operations. That is why I needed to change other places to be in sync, and made this change from `void` to `Task`.

Why do u feel bad about this?

Reply:
I see that in 2 our of 3 invocations of StartBootstrap it is awaited synchronously anyway, so I wonder why it can't be void. But given xunit supports async tests, eventually we would want to refactor it from .GetResults() to awaits anyway, so this change is fine.